packages/terra-button/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Fixed
+* Fixed Button not showing focus styles when programmatically focused
```

Comment:
What do you mean by programmatically focused?

Reply:
Programmatic focus would be getting a reference to the DOM element and calling the `focus()` method.